src/libutil/ustring.cpp

```diff
@@ -459,11 +510,8 @@ ustring::make_unique(string_view strref)
     if (!strref.data())
         strref = string_view("", 0);
 
-    hash_t hash = Strutil::strhash64(strref);
-    // This line, if uncommented, lets you force lots of hash collisions:
-    // hash &= ~hash_t(0xffffff);
+    hash_t hash = ustring::strhash(strref);
 
```

Comment:
You can use a similar "force a lot of collisions" to to locally test that the collision code is all correct, now that it's much more complicated with the ptr vs hash perfect hashing.